extension-manifest-v3/scripts/build.js

```diff
@@ -123,6 +124,43 @@ engines.forEach((engine) => {
   if (result.stderr) process.exit(1);
 });
 
+// extract resources
+shelljs.mkdir('-p', resolve(options.outDir, 'web_accessible_resources'));
+
+const seenWebAccessibleResources = new Set();
+
+FiltersEngine.deserialize(
+  readFileSync(
+    resolve(options.srcDir, 'rule_resources', `engine-ads${engineType}.dat`),
+  ),
+).resources.resources.forEach((value, key) => {
+  // Check if this is a scriptlet
+  if (
+    value.contentType === 'application/javascript' &&
+    value.body.startsWith('if(typeof scriptletGlobals==="undefined")')
+  ) {
+    return;
+  }
+
+  seenWebAccessibleResources.add('web_accessible_resources/' + key);
+
+  // Decode base64
+  if (value.contentType.endsWith(';base64')) {
+    value.body = Buffer.from(value.body, 'base64').toString('binary');
+  }
+
+  const path = resolve(options.outDir, 'web_accessible_resources', key);
+
+  shelljs.mkdir(
+    '-p',
+    dirname(resolve(options.outDir, 'web_accessible_resources', path)),
+  );
+  writeFileSync(
+    resolve(options.outDir, 'web_accessible_resources', key),
```

Comment:
lets put those is some folder, eg. `web_accessible_resource/adblocker`

Reply:
Moved to `web_accessible_resources/statics`